ikasaneip/error-reporting/src/main/java/org/ikasan/error/reporting/dao/ErrorManagementDao.java

```diff
@@ -180,4 +180,18 @@
      */
     public void updateAsHarvested(List<ErrorOccurrence> events);
 
+    /**
+     * Is harvest query ordered?
+     *
+     * @return true if ordered otherwise false.
+     */
+    public Boolean getOrderHarvestQuery();
+
+    /**
+     * Set flag to determine if harvest query should be ordered.
+     *
+     * @param orderHarvestQuery
+     */
+    public void setOrderHarvestQuery(Boolean orderHarvestQuery);
```

Comment:
If we were to use isHarvestQueryOrdered as variable 

then  following could be used as getter and setters

public void setHarvestQueryOrdered(boolean harvestQueryOrdered);

public boolean isHarvestQueryOrdered();

Reply:
> If we were to use isHarvestQueryOrdered as variable
> 
> then following could be used as getter and setters
> 
> public void setHarvestQueryOrdered(boolean harvestQueryOrdered);
> 
> public boolean isHarvestQueryOrdered();

I am removing getter but leaving the variable in its original state.